addons/hr_timesheet/security/hr_timesheet_security.xml

```diff
@@ -63,5 +63,12 @@
             <field name="groups" eval="[(4, ref('group_timesheet_manager'))]"/>
         </record>
 
+        <record id="hr_timesheet_rule_user" model="ir.rule">
+            <field name="name">hr.timesheet.rule.user</field>
+            <field name="model_id" ref="analytic.model_account_analytic_line"/>
+            <field name="domain_force">[('user_id', '=', user.id)]</field>
```

Comment:
user_id is not required on the model. Did we correctly check that we won't have an analytic line which is assigned to nobody and that would be "lost" (because invisible for all the users, except SUPERUSER_ID) ?

Reply:
Done the change